thorvg/examples/thorvg-example/main/idf_component.yml

```diff
@@ -0,0 +1,8 @@
+dependencies:
+  esp32_p4_function_ev_board_noglib:
+    version: ">=1.2.0"
+    rules:
+      - if: "target == esp32p4"
```

Comment:
What happens when this example is used for a different target?

Reply:
> What happens when this example is used for a different target?

It will compile and report an error.
Because `PPA` has been added here, and only the BSP of `esp32_p4_function_ev_board` has the `bsp_display_new_with_handles` function, so the example only supports P4.